vhdbuilder/packer/install-dependencies.sh

```diff
@@ -381,23 +381,14 @@ done
 # kubelet and kubectl
 # need to cover previously supported version for VMAS scale up scenario
 K8S_VERSIONS="
-1.17.0
-1.16.6
-1.16.4
-1.16.1
-1.14.6_v0.0.5
 1.14.7-hotfix.20200408.1
 1.14.8-hotfix.20200529.1
-1.15.3_v0.0.5
-1.15.4_v0.0.5
-1.15.5_f0.0.2
```

Comment:
not really needed for cluster/agentpool creation, kept for old vmas scale up optimization